v0/destinations/pardot/config.js

```diff
@@ -0,0 +1,29 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_URL = "https://pi.pardot.com";
```

Comment:
This end point is for production accounts. We will always take production account only?

Reply:
Yes. Probably need to change the implementation to the way `Salesforce` handles it. Good Catch 👍🏼

Reply:
As per the decision made in a meeting owing to time constraints, we will not be implementing the same currently. But it is on the road-map